src/applications/facility-locator/sass/facility-locator.scss

```diff
@@ -3,12 +3,6 @@
 @import "~leaflet/dist/leaflet.css";
 @import "~react-tabs/style/react-tabs";
 
-// TODO: These colors are not close enough to anything that already exists in
-// Formation. Perhaps the design should be reevaluated to see if Formation
-// colors could be used in place of these
-$facility-locator-color-info-bubble-border: #6495a3;
-$facility-locator-color-info-bubble-text: #818a8d;
-$facility-locator-color-gray: #ccc;
```

Comment:
The gray color wasn't used at all, and the other two were only used as descendants of the unused `#infoBubble` selector